testdata/scripts/strings.txt

```diff
@@ -141,4 +170,13 @@ skip typed const skip typed var skip typed var assign
 stringTypeField String stringTypeField strType
 stringType lambda func return
 stringType func param
-stringType return
\ No newline at end of file
+stringType return
+
+
+
+foo
+
+
+
+
```

Comment:
perhaps rewrite the test to not print every element, to avoid having so many empty lines. they're not really helping, and initially they confused me when reviewing the test.

Reply:
ok